test/oauth/testUtils.js

```diff
@@ -0,0 +1,58 @@
+const should = require('should');
+
+const services = require('../../lib/services');
+const db = require('../../lib/db');
+
+const credentialService = services.credential;
+const userService = services.user;
+const applicationService = services.application;
+
+module.exports = {
+  checkTokenResponse: (response, additionalProps = []) => {
+    should(response).have.properties('access_token', 'expires_in', ...additionalProps);
+    should(response.token_type).be.eql('Bearer');
+    should(response.access_token.length).be.greaterThan(15);
+  },
+
+  createOAuthScenario: () => {
+    const user1 = {
+      username: 'irfanbaqui',
```

Comment:
how about using a unique name? Same data across different tests can produce hard to track issues. Especially, if cleanup for some reason will not work.

Reply:
Like `uuid.v4()`?

Reply:
I've used base62 id, it is more readable. uuid is ok, probably some phrase generator like `little-brown-fox` can work better. up to you

Reply:
I'm going to use uuid62 for now, but maybe it's time to introduce https://github.com/marak/Faker.js/ — it's able to generate decent fake data as well as data based on JSON schema (that's really handy for us)

Reply:
faker looks good. but please one by one as part of other tasks. no need to refactor entire code base at one step